doc/source/whatsnew/v1.1.0.rst

```diff
@@ -148,6 +148,7 @@ Indexing
 ^^^^^^^^
 - Bug in slicing on a :class:`DatetimeIndex` with a partial-timestamp dropping high-resolution indices near the end of a year, quarter, or month (:issue:`31064`)
 - Bug in :meth:`PeriodIndex.get_loc` treating higher-resolution strings differently from :meth:`PeriodIndex.get_value` (:issue:`31172`)
+- Indexer which is list of string has a datetime format fails on :class:`PeriodIndex` or :class:`DatetimeIndex` (:issue:`11278`)
```

Comment:
fails -> failed
"which is list of string has a datetime format" -> "with a list of strings representing datetimes"